etc/profile-m-z/ouch.profile

```diff
@@ -0,0 +1,13 @@
+# Firejail profile for ouch
+# Description: CLI utility for easily compressing and decompressing files and dirs
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include ouch.local
+# Persistent global definitions
+include globals.local
+
+noroot
```

Comment:
```suggestion
```

This is commented in archiver-common, probably to avoid issues with group
permissions.